src/uct/api/v2/uct_v2.h

```diff
@@ -174,6 +177,18 @@ typedef struct {
      * This field is set by the UCT layer.
      */
     ucs_linear_func_t   latency;
+
+    /**
+     * Approximated maximum number of endpoints that could have outstanding
+     * operations simultaneously. If additional endpoints are being used for
+     * sending operations, these operations will most likely return
+     * UCS_ERR_NO_RESOURCE.
```

Comment:
i'd removed the sentence about `NO_RESOURCE`, it's mostly about local vs remote completions as discussed offline

Reply:
ok